system/Test/bootstrap.php

```diff
@@ -16,12 +16,14 @@
 defined('APPPATH')       || define('APPPATH', realpath($paths->appDirectory) . DIRECTORY_SEPARATOR);
 defined('WRITEPATH')     || define('WRITEPATH', realpath($paths->writableDirectory) . DIRECTORY_SEPARATOR);
 defined('SYSTEMPATH')    || define('SYSTEMPATH', realpath($paths->systemDirectory) . DIRECTORY_SEPARATOR);
+defined('VIEWPATH')      || define('VIEWPATH', realpath($paths->viewDirectory) . DIRECTORY_SEPARATOR);
```

Comment:
VIEWPATH isn't defined in `system/bootstrap` so don't define it here. See other comments.